docs/navigation/cell.mdx

```diff
@@ -1 +1,135 @@
-cell page
\ No newline at end of file
+---
+product: dd360-ds
+title: React Cell component
+components: Cell
+---
+
+<br />
+<br />
+
+# <span name="floating-nav">Cell</span>
+
+With the Cell component you can render a table cell with customizable style and behavior.
```

Comment:
Mmmm, this is not `Cell` of the table @arielsrodriguez 
I supose when you talk about `Cell` you reference to `Table.Cell` and not is the same component.

Do you can re-check this component on storybook to improve the description?